### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by BasilisKing**
Assignment 4 and corrections on assignments 2 & 3

### CODE REVIEW

**Comment by dachafra** [comment]
this PR also contains assign2 and assign3, 

Assign4 is ok.
there is a minor issue in task06, vcard:University does not exist ;-)
The PR is going to be merged although there is that minor error.

**Review by paoespinozarias — COMMENTED** [review]
Assignment 2b is ok

File: Assignment2/BasilisKing-DanielCanada.ttl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+cl:Class1 class:includes s:Sensor029;
+          class:includes c:Computer101.
+
+s:Sensor029 sensor:hasMeasurement m:Measurement8401.
+
+c:Computer101 comp:hasOwner o:User10A.
+
+m:Measurement8401 mes:hasTemperature "29"^^xsd:Integer;
+                  mes:atTime "2010-06-12T12:00:12".
```
Comment (ocorcho) [inline Assignment2/BasilisKing-DanielCanada.ttl:22]: This should be datatyped as well.

File: Assignment2/BasilisKing-DanielCanada.ttl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@prefix own: <http://www.example.org/ontologies/owner#>
+
+cl:Class1 class:includes s:Sensor029;
+          class:includes c:Computer101.
+
+s:Sensor029 sensor:hasMeasurement m:Measurement8401.
+
+c:Computer101 comp:hasOwner o:User10A.
+
+m:Measurement8401 mes:hasTemperature "29"^^xsd:Integer;
```
Comment (ocorcho) [inline Assignment2/BasilisKing-DanielCanada.ttl:21]: Apply a correct datatype. See what others have done.

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Assigment 2a still needs to be fixed. Assignment 3 is ok

**Pull request merged**